lab/lab2/part3-underscore-refactor.js

```diff
@@ -95,9 +95,10 @@
 
   function styler (filtered) {
```

Comment:
I know that everything up until this point has worked, but the final markers aren't showing up on the map. I can't figure out if it's something wrong with this styler function or the last line of code (the main loop)